src/Query/Builder.php

```diff
@@ -890,6 +890,33 @@ public function decrementEach(array $columns, array $extra = [], array $options
         return $this->incrementEach($decrement, $extra, $options);
     }
 
+    /** @inheritdoc */
+    public function multiply($column, $amount = 1, array $extra = [], array $options = [])
```

Comment:
I don't find this method in the parent class. I only find [`Illuminate\Collections\Collection::multiply`](https://github.com/laravel/framework/blob/bcc92201ab6d786477fed578cc2e3ea4c4e1b093/src/Illuminate/Collections/Collection.php#L881). 

Could you remove the `@inheritdoc` annotation and add a description for all `@param` and `@return`.

Reply:
Done 👌🏻